tests/layer_tests/tensorflow_tests/test_tf_Sub.py

```diff
@@ -7,212 +7,364 @@
+#                 **params, ir_version=ir_version, use_legacy_frontend=use_legacy_frontend
+#             ),
+#             ie_device,
+#             precision,
+#             ir_version,
+#             temp_dir=temp_dir,
+#             use_legacy_frontend=use_legacy_frontend,
+#         )
+
+#     test_data_broadcast_4D = [
+#         dict(x_shape=[1, 1, 1, 1], y_shape=[1]),
+#         dict(x_shape=[1, 3, 1, 1], y_shape=[1]),
+#         dict(x_shape=[1, 3, 1, 1], y_shape=[3]),
+#         dict(x_shape=[1, 100, 224, 3], y_shape=[3]),
+#         dict(x_shape=[1, 1, 1, 3], y_shape=[3]),
+#         dict(x_shape=[1, 3, 1, 1], y_shape=[3, 1]),
+#         dict(x_shape=[1, 3, 1, 2], y_shape=[3, 1, 2]),
+#         dict(x_shape=[1, 3, 1, 2], y_shape=[1, 3, 2]),
+#         dict(x_shape=[1, 3, 100, 224], y_shape=[1, 1, 1, 224]),
+#         dict(x_shape=[2, 3, 1, 2], y_shape=[1, 3, 2, 1]),
+#     ]
+
+#     # TODO mark as precommit (after successfully passing in nightly)
+#     @pytest.mark.parametrize("params", test_data_broadcast_4D)
+#     @pytest.mark.nightly
+#     def test_sub_placeholder_const_broadcast_4D(
+#         self, params, ie_device, precision, ir_version, temp_dir, use_legacy_frontend
+#     ):
+#         self._test(
+#             *self.create_sub_placeholder_const_net(
+#                 **params, ir_version=ir_version, use_legacy_frontend=use_legacy_frontend
+#             ),
+#             ie_device,
+#             precision,
+#             ir_version,
+#             temp_dir=temp_dir,
+#             use_legacy_frontend=use_legacy_frontend,
+#         )
+
+#     test_data_broadcast_5D = [
+#         dict(x_shape=[1, 1, 1, 1, 1], y_shape=[1, 1, 1, 1, 1]),
+#         dict(x_shape=[1, 3, 1, 1, 1], y_shape=[1, 1]),
+#         dict(x_shape=[1, 3, 1, 1, 1], y_shape=[3]),
+#         dict(x_shape=[1, 1, 1, 1, 3], y_shape=[3]),
+#         dict(x_shape=[1, 3, 1, 1, 1], y_shape=[3, 1]),
+#         pytest.param(
+#             dict(x_shape=[1, 3, 1, 1, 2], y_shape=[1, 3, 2]),
+#             marks=pytest.mark.precommit_tf_fe,
+#         ),
+#         dict(x_shape=[1, 3, 5, 1, 2], y_shape=[3, 5, 1, 2]),
+#         dict(x_shape=[1, 3, 50, 100, 224], y_shape=[1, 1, 1, 1, 224]),
+#         dict(x_shape=[2, 3, 1, 2, 1], y_shape=[1, 3, 2, 1, 1]),
+#     ]
+
+#     # TODO mark as precommit (after successfully passing in nightly)
+#     @pytest.mark.parametrize("params", test_data_broadcast_5D)
+#     @pytest.mark.nightly
+#     def test_sub_placeholder_const_broadcast_5D(
+#         self, params, ie_device, precision, ir_version, temp_dir, use_legacy_frontend
+#     ):
+#         self._test(
+#             *self.create_sub_placeholder_const_net(
+#                 **params, ir_version=ir_version, use_legacy_frontend=use_legacy_frontend
+#             ),
+#             ie_device,
+#             precision,
+#             ir_version,
+#             temp_dir=temp_dir,
+#             use_legacy_frontend=use_legacy_frontend,
+#         )
```

Comment:
please do not comment the current test. Just Add separate test suite `TestComplexSub`

Reply:
@rkazants 
My apologies, I've reverted that change, and added TestComplexSub